providers/common/ciphers/cipher_aes_gcm_hw_s390x.inc

```diff
@@ -97,10 +97,7 @@ static int s390x_aes_gcm_cipher_final(PROV_GCM_CTX *ctx, unsigned char *tag)
         memcpy(tag, kma->t.b, ctx->taglen);
         rc = 1;
     } else {
-        if (ctx->taglen < 0)
```

Comment:
This is not quite right..
It should now be checking  if (ctx->taglen == UNINITIALISED_SIZET) rc = 0; else

Reply:
Can I suggest we check for such error much higher in the call stack?

Reply:
will do - thanks for the suggestion..